apps/admin/frontend/src/app.test.tsx

```diff
@@ -629,32 +533,6 @@ test('election manager cannot auth onto machine with different election hash', a
   );
 });
 
-test('primary election flow', async () => {
```

Comment:
the only purpose of this test was to test L&A documents for a primary settings, so it's also no longer necessary